docs/download.md

```diff
@@ -16,12 +16,17 @@ Please, create a fork to edit the code or use the GitHub [issue tracker](https:/
 </div>
 <div>
 [<button type="button" class="btn btn-success">
-**MacOS** <img src="../img/apple-white.png" alt="apple logo" height="25">
+**MacOS (in progress)** <img src="../img/apple-white.png" alt="apple logo" height="25">
```

Comment:
coming soon ?

Reply:
I everything goes well, at the end of this build: https://ci.appveyor.com/project/ringmesh/ringmesh/build/953

Reply:
No, I think probably never since it is a geogram issue that will never be fixed by its author.. @benjaminchauvin is supposed to look at this issue, if he can modify geogram cmake files to fix the problem.

Reply:
But keeping the button in progress is interesting to make tests and be able to download the current release. At least until we completly drop MacOS binary support or make it work.